### ISSUE TITLE
Variables in multiple tables with different types will cause a clash

### ISSUE THREAD

**Issue by JackWilb** [multinet-app/multilink#176]
Currently, all code related to types implicitly expects each variable name to be unique for the type mapping. This assumption is probably not true, and in the case where we have the same variable name in different tables with different types, we'll have an issue.

**Comment by waxlamp** [comment]
Fixing this could be as simple as keeping track of `<table name>/<column name>` instead of just `<column name>`, right?

**Comment by JackWilb** [comment]
Yes. It would require changing to that around the app though so that we're always passing in that format
